### DISCUSSION TITLE
[Question] Is it possible to add a post deployment hook?

### DISCUSSION THREAD

**Discussion by NanoCode012** [vercel/community#758] [Help] [answered]
Hello!

I was wondering if it's possible to add a post deployment hook for Vercel deployments? My usecase is to run a specific script after the build is complete. 

I connected Vercel via Github, if that could help. Perhaps, is there a way to inform the deployment is completed to Github to run Github actions?

I searched for keywords, but did not come across any on the Discussions here. I only found a SO post with zero answers https://stackoverflow.com/questions/64590841/vercel-running-a-node-script-after-deployment .

**Comment by swarnava** [comment, marked as answer]
What sort of task are you planning to do? There are couple of ways to do it. You can either configure them in [Build Steps](https://vercel.com/docs/concepts/deployments/configure-a-build) or use GitHub Action to setup your deployment (instead of native GitHub integration).

**Reply by NanoCode012** [reply]
@swarnava Thank you very much for your answer. These do look like potential options. I also found https://github.com/UnlyEd/github-action-await-vercel. 

My use case is to apply some SQL via `psql`. I've been trying to use Prisma with Supabase. However, due to a limitation of Prisma only working on a single schema, I need to apply all other schema SQL via an external `psql` after Prisma migrations. 

Currently, I am evaluating my workflow again and trying to see if I actually need the hook, however, thanks again for your ideas!
